demos/simple_jump_bot/README.md

```diff
@@ -0,0 +1,22 @@
+# HammettSimpleJumpBot
+
+This demo showcases the `move` and `jump` features provided by Hammett. `move` allows you to navigate between different screens while re-rendering the same message, and `jump` lets you send screens as a new message, saving the previous ones in the chat history.
+
+To use it, you need to create a keyboard with a button which source type is `JUMP_SOURCE_TYPE` and a source is a screen which you want to be sent as new message. Well, you can see the implementation in the `demo.py` module.
```

Comment:
Remove this sentences in favor of `Check out the documentation for more details.` (see :point_up_2: )